src/plugins/orangehrmBuzzPlugin/Api/BuzzPostAPI.php

```diff
@@ -262,39 +263,59 @@ private function getCommonBodyValidationRules(): array
      */
     private function getPhotoValidationRule(): ParamRule
     {
+        $postType = $this->getRequestParams()->getString(
```

Comment:
reason??
also did you covered tests, if the request body does not contains post type parameter